hudi-common/src/test/java/org/apache/hudi/common/testutils/SchemaTestUtil.java

```diff
@@ -66,12 +66,12 @@ public static Schema getSimpleSchema() throws IOException {
       throws IOException, URISyntaxException {
     GenericDatumReader<IndexedRecord> reader = new GenericDatumReader<>(writerSchema, readerSchema);
     // Required to register the necessary JAR:// file system
-    URI resource = SchemaTestUtil.class.getClass().getResource("/sample.data").toURI();
+    URI resource = SchemaTestUtil.class.getResource("/sample.data").toURI();
```

Comment:
It would be better to extract those file names into constant fields in this file. WDYT?

Reply:
yup fixing it